### PR TITLE
[US-395] Add example code for multifont encoder

### PR DESCRIPTION
**Pull request by kellemNegasi**

### CODE REVIEW

File: multi-font-encoder/render_multi_language_text.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	if err != nil {
+		panic(err)
+	}
+	if err := c.WriteToFile("multiple-language-text-multi-font.pdf"); err != nil {
+		panic(err)
+	}
+}
+
+// getFonts returns list of *model.PdfFont from list of paths to fonts files.
+func getFonts(fontPaths []string) []*model.PdfFont {
```
Comment (sampila) [inline multi-font-encoder/render_multi_language_text.go:50]: Probably this name will be better `setCompositeTTFPdfFonts?`

**Review by sampila — COMMENTED** [review]
Overall looks good, added minor comment

**Review by 3ace — APPROVED** [review]
lgtm

File: multi-font-encoder/render_multi_language_text.go (hunk 1 above)
Comment (kellemNegasi) [reply multi-font-encoder/render_multi_language_text.go:50]: Yea, that seems more expressive. may be `useCompositeTTFPdfFonts`

**Review by anovik — APPROVED** [review]

**Pull request merged**